README.md

```diff
@@ -87,6 +87,7 @@ SC-machine is a core of any OSTIS-system, so you can use a reference system name
   ```
 
 Most of these commands have a help page bundled inside, so if you have any questions or want to customize the command behavior, use `--help` flag to learn more about them.
+You can also use `./scripts/build_kb.sh` and `./scripts/run_sc_server.sh` to do it. But you need to configure environment variables in `./scripts/set_vars.sh` and call it.
```

Comment:
It seems to me that it will be more user-friendly to use scripts as the main way to build kb and launch sc-server and describe the binaries themselves as an alternative, you just have to type less that way 🤣. The piece about `set_vars.sh` is written in a very ambiguous way. I would do smth like "Note that for projects other than sc-machine you might need to copy set_vars.sh to your project, change the variables accordingly and call it before using the scripts provided by sc-machine". You can paraphrase it in a more concise way perhaps, but the clarity should be saved.

Reply:
Ok, half-resolved. Feel free to resolve if you think that binaries should be the default option in the guide and scripts are an alternative and not vice-versa